### DISCUSSION TITLE
Some tutorials

### DISCUSSION THREAD

**Discussion by madukan** [americanexpress/unify-flowret#23] [Ideas] [answered]
Hi Deepak,

Are there any tutorials, good videos at the moment?
Something simple and a bit complex one (single flow, and a parallel).
I’m after the steps + sample code would be great.

**Comment by deepakarora3** [comment]
Hi - thanks for reaching out and showing interest. Actually, the documentation I have written I think is quite exhaustive and so I am hoping that would be sufficient. Please let me know what challenges you face, and I can try and address those. As regards sample code, you will find plenty of examples in the tests folder for both single threaded and parallel processing. Thanks.

**Comment by madukan** [comment]
It is for sure. Just to get started, it was a bit tricky. Lets say for
example, the unit tests, there seem tests that can be tested the framework.
So is good for a developer who upgrade Flowret. However lets say a
"user-developer", who like to do a simple "Hello world" to get something
working, it seem a bit tricky. Ideally within 5-10 mins for them to get
going. Small tutorial. Would be ideal.

**Reply by deepakarora3** [reply, marked as answer]
Thanks! Let me put a section on "Getting started in 5 min". It may take a week or two but I will get there :)

**Comment by madukan** [comment]
Sounds good. Thanks. Give a shout if I can help with anything.

**Comment by deepakarora3** [comment]
Hi - its been a few months but finally it is here - getting started in 5 minutes - sample program in code and explained in the documentation. Just wanted to let you know that it is there now.
